lib/main.rb

```diff
@@ -32,7 +32,7 @@ def download(array_of_files)
         s3.download_file_by_name(filename, "#{@tmp_dir}/#{dir_name}")
       rescue StandardError
         p("Failed download:#{filename}")
-        logger("#{@tmp_dir}/Failed_download_log.txt", "Failed download: #{filename}\n", 'a')
+        Helper.file_writer("#{@tmp_dir}/Failed_download_log.txt", "Failed download: #{filename}\n", 'a')
```

Comment:
Read about `logger` classes in ruby - I think this can be replaced on 
`logger.info`